### ISSUE TITLE
[Broken Website] blue flash with discourse forums

### ISSUE THREAD

**Issue by B0pol** [darkreader/darkreader#2677]
**Website**
https://discussion.fedoraproject.org/
https://community.e.foundation
it's a forum software and hosted in may websites
**How to Reproduce**
1. Open a post
**Expected behavior**
I'm not blinded because a blue flash
**Actual behavior**
A blue flash kill your eyes

**System info:**
 - OS: fedora 32, android 9
 - Browser  firefox 76.0.1, firefox preview 5.0
 - Darkreader Version 4.9.9

**Additional context**
I don't know if it can be fixed by darkreader since it's not about a precise website but a software hosted on many different websites.
if it isn't, i'll report it to discourse

**Comment by Gusted** [comment]
Could you define 'blue flash'

**Comment by B0pol** [comment]
![ScreenRecord-2020-05-31-16-54-47](https://user-images.githubusercontent.com/58657617/83355433-bb6e2580-a35f-11ea-9d50-beb720f8033d.gif)

**Comment by Gusted** [comment]
Seems like it could they use `@keyframes`

#2590

**Comment by Gusted** [comment]
Yes it's a `@keyframe` So wait until the PR is merged and next release 😬 

```
@keyframes background-fade-highlight { 
    0% { 
        background-color: rgb(209, 240, 255);
    }  
    100% { 
        background-color: transparent; 
    }
}
```

**Comment by B0pol** [comment]
Amazing, thanks for you work
